crates/runtime/src/http/v1/eval.rs

```diff
@@ -103,6 +103,7 @@ pub(crate) async fn post(
 
     let evals = rt.evals.read().await;
     let Some(eval) = evals.iter().find(|e| e.name == eval_name) else {
+        tracing::error!("Failed to run the eval `{eval_name}`. The eval does not exist.\nVerify the configuration and try again.");
```

Comment:
I'm not sure if it makes sense for an API request that has user-controlled input to be able to pollute the error logs of spice

Reply:
I agree